L/libpolymake_julia/common.jl

```diff
@@ -5,11 +5,12 @@ import Pkg: PackageSpec
 import Pkg.Types: VersionSpec
 
 name = "libpolymake_julia"
-version = v"0.3.0"
+version = VersionNumber(0, 4, julia_version.minor)
+upstream_version = v"0.3.0"
 
 # Collection of sources required to build libpolymake_julia
 sources = [
-    ArchiveSource("https://github.com/oscar-system/libpolymake-julia/archive/v$(version).tar.gz",
+    ArchiveSource("https://github.com/oscar-system/libpolymake-julia/archive/v$(upstream_version).tar.gz",
                   "937d1a6e4d8146bb92e8e71c0fad4a8de782362c9b4e51ced3158aab908b1e07"),
```

Comment:
```suggestion
    ArchiveSource("https://github.com/oscar-system/libpolymake-julia/archive/v$(upstream_version).tar.gz",
                  "004380813658ba218092ba526a8acf884796fe9afa581552523bd302210293af"),
```
We could also switch to the corresponding commit `54989d411333fb94591406084327e8489ad665b2`, I don't really care whether we use archive or git.